code/datums/mutations/speech.dm

```diff
@@ -120,27 +120,49 @@
 	var/message = speech_args[SPEECH_MESSAGE]
 	if(message)
 		message = " [message] "
+		message = replacetext(message," security ",pick(" coppers "," bobbies "))
+		message = replacetext(message," friend ",pick(" mate "," bruv "," lad "))
+		message = replacetext(message," great ",pick(" bangin' "," sound "))
+		message = replacetext(message," man ",pick(" mate "," mans "))
+		message = replacetext(message," isnt ",pick(" innit "," aint it "))
+		message = replacetext(message," dumb ",pick(" daft "," stupid "))
+		message = replacetext(message," idiot ",pick(" wanker "," tosser "," mong "," prick "))
```

Comment:
one of these will get us pinged by github ToS

Reply:
I removed it, and I'm not sure how exactly you mean to put it in a config (I was originally gonna remove it because the people I was writing the PR with thought it wasnt good, but forgot to actually remove it). You want the .json to just be under the config folder?